cmd/dep/init.go

```diff
@@ -192,14 +192,9 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 			}
 		}
 		if newProject {
-			// Check if it's in notondisk project map. These are direct deps, should
-			// be added to manifest.
+			// If it's in notondisk, add to manifest, these are direct dependencies.
 			if _, ok := pd.notondisk[pr]; ok {
 				m.Dependencies[pr] = getProjectPropertiesFromVersion(x.Version())
-				feedback(x.Version(), pr, fb.DepTypeDirect, ctx)
```

Comment:
Now that we don't log feedback for new projects found by the solver, let's update the comment on the for loop to remove `if direct deps and log feedback for all the new projects.`.